packages/create-instantsearch-app/src/cli/index.js

```diff
@@ -159,6 +164,36 @@ const getQuestions = ({ appName }) => ({
       when: ({ appId, apiKey, indexName }) =>
         attributesToDisplay.length === 0 && appId && apiKey && indexName,
     },
+    {
+      type: 'list',
+      name: 'imageAttribute',
+      message: 'Attribute for image display',
+      suffix: `\n  ${chalk.gray(
+        'Used to display images in the default result template'
+      )}`,
+      pageSize: 10,
+      choices: async (answers) => [
+        {
+          name: 'None',
+          value: undefined,
+        },
+        new inquirer.Separator(),
+        new inquirer.Separator('From your index'),
+        ...(await getPotentialImageAttributes(answers)),
```

Comment:
does inquirer have a "text input" as fallback possible or not?

Reply:
Not from what I've seen, we would need to have a `manual` option that would trigger another step but I'm not sure it's worth the trouble